DAOS-Support.md

```diff
@@ -10,22 +10,25 @@ filesystems:
 2. POSIX -> DAOS
 3. DAOS  -> DAOS
 
+The DAOS->DAOS case supports both POSIX and non-POSIX DAOS containers.
+
 ## DAOS Data Movement Use Cases
 
-In each use case, it is assumed the the pools used already exist. Also,
-only one DAOS source is supported.
+In each use case, it is assumed the the pools and containers used already exist.
+Also, only one DAOS source is supported.
 
 1. **DAOS Destination**
-    * POSIX container may or may not exist already
+    * Container is assumed to exist already
```

Comment:
@dsikich If this is the case, should we remove the auto-creation of the destination?

Reply:
We could do that in a different PR

Reply:
@daltonbohning yes, probably. It was removed from the other tools since it could cause creating a lot of unnecessary destination containers if the user just forgot to specify the destination or something

Reply:
@dsikich I'll create a ticket to keep track of this